addons/skin.confluence/720p/DialogKeyboard.xml

```diff
@@ -20,6 +20,54 @@
 				<param name="CloseButtonLeft" value="770" />
 				<param name="CloseButtonNav" value="3" />
 			</include>
+			<control type="group">
+				<left>0</left>
+				<top>-90</top>
+				<visible>Control.IsVisible(313)</visible>
```

Comment:
agreed. it would be better if we had an infobool we could check instead.
something like Keyboard.HasCodingInput or similar.